Automattic/wp-calypso
# OAuth: Customise login pages for OAuth clients

Automattic OAuth clients require the ability to customise the layout for their login page.

#### Testing instructions

1. Run `git checkout add/login-oauth-layout` and start your server, or open a [live branch](https://calypso.live/?branch=add/login-oauth-layout)
2. Open the [`OAuth Log In` page](http://calypso.localhost:3000/log-in?client_id=1)
3. Confirm that the logged out Master Bar is removed
4. Confirm that the Back link at the bottom points to the app page
5. Confirm that `document.getElementsByClassName('dops');` is empty
5. Repeat 2-4 with `client_id=973`
6. Confirm that `document.getElementsByClassName('dops');` returns an element

#### Reviews
